app/javascript/widget/store/modules/campaign.js

```diff
@@ -29,10 +28,7 @@ export const actions = {
       campagins: formattedCampaigns,
       currentURL,
     });
-    campaignTimer.init({
-      allCampaigns: filteredCampaigns,
-      currentURL,
-    });
+    campaignTimer.initTimers({ allCampaigns: filteredCampaigns });
```

Comment:
One small feedback on the naming. Shouldn't it be enough to have the variable named as `campaigns` rather than `allCampaigns`